src/gridcoin/beacon.h

```diff
@@ -910,6 +933,13 @@ class BeaconRegistry : public IContractHandler
         //!
         uint64_t m_recnum_stored = 0;
 
+        //!
+        //! \brief The flag that indicates whether memory optimization can occur by passivating the database. This
+        //! flag is set true when find() or operator[] retrieves a beacon element from leveldb to satisfy a hash search.
+        //! This would typically occur on a beacon renewal or reorganization (revert).
```

Comment:
```suggestion
        //! flag is set true when find() retrieves a beacon element from leveldb to satisfy a hash search.
        //! This would typically occur on a beacon renewal or reorganization (revert).
```